test/fixtures/output/network/describe.golden

```diff
@@ -1,20 +1,12 @@
-| Display Name             | prod-aws-us-east1                                                                                              |
-| Cloud                    | AWS                                                                                                            |
-| Region                   | us-east-1                                                                                                      |
-| ConnectionTypes          | {[PRIVATELINK]}                                                                                                |
-| Cidr                     | 10.200.0.0/16                                                                                                  |
-| Zones                    | [use1-az1 use1-az2 use1-az3]                                                                                   |
-| DnsResolution            | CHASED_PRIVATE                                                                                                 |
-| Phase                    | READY                                                                                                          |
-| SupportedConnectionTypes | {[PRIVATELINK]}                                                                                                |
-| ActiveConnectionTypes    | {[PRIVATELINK]}                                                                                                |
-| ReservedCidr             | 172.20.255.0/24                                                                                                |
-| Resource URL             | https://api.confluent.cloud/networking/v1/networks/n-abcde1                                                    |
-| Resource Name            | crn://confluent.cloud/organization=9bb441c4-edef-46ac-8a41-c49e44a3fd9a/environment=env-00000/network=n-abcde1 |
-| Created At               | 2023-02-24T00:00:00Z                                                                                           |
-| Updated At               | 2023-02-24T00:00:00Z                                                                                           |
-| Deleted At               | 2023-02-24T00:00:00Z                                                                                           |
-+--------------------------+----------------------------------------------------------------------------------------------------------------+
++-------------------------+------------------------------+
+| ID                      | n-abcde1                     |
+| Environment ID          | env-00000                    |
+| Name                    | prod-aws-us-east1            |
+| Cloud                   | AWS                          |
+| Region                  | us-east-1                    |
+| CIDR                    | 10.200.0.0/16                |
+| Zones                   | [use1-az1 use1-az2 use1-az3] |
```

Comment:
Can you make a second parallel output struct where this is a comma-delimited string?

Reply:
Are you referring to `Zones`? I see that other commands have the same format as well (e.g. https://github.com/confluentinc/cli/blob/traffic-8850-network-cli/test/fixtures/output/kafka/topic/describe-topic-success.golden). Do we want to make it as a comma-delimited string?

Reply:
I changed the type for `Zones` and use `strings.Join(network.Spec.GetZones(), ",")` instead of making a second parallel output struct, is this okay?

Reply:
Usually we make a parallel struct so the serialized output is in list format. If users want to use this command in a script, then they won't have to call `split()` on a comma-delimited string that way.

Reply:
Is there an example in the codebase that I can use as a reference?

Reply:
https://github.com/confluentinc/cli/blob/c0a3c77efa635813ea144331b1f9775553d1de45/internal/pipeline/command.go#L16-L40 this was the reference sent by Brian on slack.

Reply:
@brianstrauch just updated the code!